File: examples/missing/example_impute1.R
library("mmstat4")
body <- ghload("data/allbus2012.rds")
body <- body[,c("age", "height", "weight")]
# number of NAs
nabody <- is.na(body)
apply(nabody, 2, sum)
# full data
mean(body$weight)
cor(body)
# case deletion
mean(body$weight, na.rm=T)
cor(body, use="complete.obs")
sum(complete.cases(body))
# available case analysis
cor(body, use="pairwise.complete.obs")
crossprod(!nabody)

File: examples/missing/example_impute1.py
import pandas as pd, numpy as np
body = r["mmstat4::ghload"]("allbus2012.rds")
body = body[["age", "height", "weight"]]
# Number of NAs
nabody = body.isna()
nabody_sum = nabody.sum(axis=0)
print(nabody_sum)
# Default: case deletion
print(body["weight"].mean())
print(body.corr())
